docs-2.0/graph-computing/0.deploy-controller-analytics.md

```diff
@@ -124,7 +124,17 @@ There are some differences between installation packages and commands in differe
     VarDataListMaxSize: 100    # If HDFS columns are read, the number is limited to 100 at a time.
   ```
 
-5. Configure the algorithm file path (`exec_file`) in the file `tasks.yaml`, the file path is `dag-ctrl/etc/tasks.yaml`. If there are multiple machines, ensure that the paths are the same.
+5. Configure the algorithm file path (`exec_file`) only in the file `tasks.yaml`, the file path is `dag-ctrl/etc/tasks.yaml`. Currently, all `exec_file` parameters are set to the path of the `run_algo.sh` file.
```

Comment:
the file path is `dag-ctrl/etc/tasks.yaml`.

the file path ---> which file do you specify? as to this sentence, it seems to be the exec_file.